src/main/java/cinemas/enums/MovieStatus.java

```diff
@@ -1,7 +1,22 @@
 package cinemas.enums;
 
 public enum MovieStatus {
-    NOW_SHOWING,
-    COMING_SOON,
-    END_SHOWING
+    NOW_SHOWING("Now Showing", "Đang chiếu"),
```

Comment:
không sửa ở đây được, lưu status vô db là các số int

Reply:
suggest dùng MovieCreateFormDto để binding và validate form, sau đó truyền form này vào service và mapping vào Movie. tham khảo https://github.com/awesome-academy/natiei2024-batch-1-website-for-booking-tickets-and-managing-cinemas/blob/master/src/main/java/cinemas/controllers/admin/ScreensController.java